tests/sample/map.c

```diff
@@ -18,6 +18,12 @@
 
 #include "bpf_helpers.h"
 
+// https://github.com/vbpf/ebpf-verifier/issues/427
+// An issue in the ebpf-verifier causes it to fail on bpf_printk format strings.
+// Until that is fixed, defined bpf_printk as a no-op.
```

Comment:
Should we also file a bug in our repo to revert this change later and mark it blocked on verifier bug?

Reply:
Files https://github.com/microsoft/ebpf-for-windows/issues/1715 to track restoring this.